Tasks/Common/clienttool-common/ClientToolRunner.ts

```diff
@@ -0,0 +1,67 @@
+let fs = require("fs");
```

Comment:
Same here - https://github.com/microsoft/azure-pipelines-tasks/blob/585663dc99b62b5ab75723091d639eaf77bfade6/common-npm-packages/packaging-common-v3/universal/ArtifactToolRunner.ts

Reply:
Synced offline. Can you add a reference to the packaging-common/ArtifactToolRunner.ts in this file and vice versa, so that we know this code is duplicated and we the developer needs to update at both the places.

Similarly for ClientToolUtilities